stores/cloudflare/src/storage/index.ts

```diff
@@ -90,6 +90,8 @@ export class CloudflareStore extends MastraStorage {
   public get supports() {
     const supports = super.supports;
     supports.getScoresBySpan = true;
+    supports.resourceWorkingMemory = true;
+    supports.selectByIncludeResourceScope = true;
     return supports;
```

Comment:
turns out all the storage adapters support resource scope but it just wasn't toggled on